torch/csrc/jit/interpreter.cpp

```diff
@@ -717,6 +720,14 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     return false;
   }
 
+  void handleError(const std::string& error_msg) {
```

Comment:
nit: this sequence of calls have 2--3 unnecessary string copies in it because the error message is not being std::move'd